bin/process-invites

```diff
@@ -1,11 +1,11 @@
 #!/usr/bin/env node
 
-const getUsersCount = require('../db-helpers').getUsersCount,
- getInvitees = require('../get-invitees'),
- inviteAndFollowUsers = require('../invite-and-follow-users')
+const getMostRecentDateSubmit = require('../db-helpers').getMostRecentDateSubmit,
+    getLatestResponses = require('../typeform-helpers').getLatestResponses,
+    inviteAndFollowUsers = require('../invite-and-follow-users')
 
-getUsersCount( function (numUsers) {
-    getInvitees( numUsers, function (users) {
+getMostRecentDateSubmit( (lastSubmitTimeStamp) => {
+    getLatestResponses(lastSubmitTimeStamp, (users) => {
```

Comment:
Instead of processing invites by first counting the rows in the db and adding that to an offset, we instead get the most recent `date_submit` in the db and then get all responses  (if any) submitted after that date.